### PR TITLE
Feature/gh deploy

### PR DESCRIPTION
**Pull request by DavidMinarsch**
- Tried to address the warnings gatsby was showing (taught me a bit about the system, feel free to revert)
 -  Added support for github deployment

### CODE REVIEW

File: web/gatsby-browser.js [hunk 1]
```diff
@@ -5,17 +5,17 @@
...
- export const onRenderBody = ({ setPreBodyComponents }) => {
-   setPreBodyComponents([
-     <ColorModeScript
-       initialColorMode={theme.config.initialColorMode}
-       key="chakra-ui-no-flash"
-     />
-   ])
- }
\ No newline at end of file
+ // export const onRenderBody = ({ setPreBodyComponents }) => {
```
Comment (DavidMinarsch) [inline web/gatsby-browser.js:14]: This wasn't working locally; says API is no longer active...

File: web/gatsby-config.js [hunk 2]
```diff
@@ -5,7 +5,6 @@ module.exports = {
     author: `Team AEA`,
   },
   plugins: [
-    `@chakra-ui/gatsby-plugin`,
```
Comment (DavidMinarsch) [inline web/gatsby-config.js:8]: Removed as not used and throwing an error for me

File: web/package.json [hunk 3]
```diff
@@ -45,7 +45,8 @@
     "start": "npm run develop",
     "serve": "gatsby serve",
     "clean": "gatsby clean",
-    "test": "echo \"Write tests! -> https://gatsby.dev/unit-testing\" && exit 1"
+    "test": "echo \"Write tests! -> https://gatsby.dev/unit-testing\" && exit 1",
+    "deploy": "gatsby build && gh-pages -d public"
```
Comment (DavidMinarsch) [inline web/package.json:49]: added

File: web/src/components/footer.js [hunk 4]
```diff
@@ -12,12 +12,12 @@ const Footer = () => (
...
             <Text>© AEA {new Date().getFullYear()}</Text>
             <Link as={GatsbyLink} to='/blog'>Blog</Link>
-            <Link href="https://docs.fetchai.com" isExternal href='https://fetch.ai'>Docs</Link>
+            <Link isExternal href='https://docs.fetch.ai'>Docs</Link>
           </HStack>
         </Flex>
       </Box>
       <HStack spacing={6}>
-        {/* <Link to="https://twitter.com/"><FaTwitter /></Link> */}
+        <Link isExternal href="https://twitter.com/"><FaTwitter /></Link>
```
Comment (DavidMinarsch) [inline web/src/components/footer.js:20]: Will create account tomorrow

**Review by oaksprout — APPROVED** [review]
I took a quick look over this and there were a couple things which I wanted to pull and test, but just don't have the bandwidth in these days. I'd suggest I approve the request and if everything looks like it's running well for you, you go ahead and merge. Once a deployment is up maybe you can send me the link and I'll do some quick Q&A to make sure there's nothing major broken!

**Pull request merged**